src/zeit/content/article/edit/volume.py

```diff
@@ -0,0 +1,47 @@
+from zeit.cms.i18n import MessageFactory as _
+import grokcore.component as grok
+import zeit.cms.content.reference
+import zeit.content.article.edit.interfaces
+import zeit.content.article.edit.reference
+import zeit.content.volume.interfaces
+
+
+class ReferenceProperty(zeit.cms.content.reference.SingleReferenceProperty):
+
+    def __set__(self, instance, value):
+        saved_attributes = {name: getattr(instance, name) for name in [
+            '__name__',
+        ]}
+
+        super(ReferenceProperty, self).__set__(instance, value)
+
+        for name, val in saved_attributes.items():
+            setattr(instance, name, val)
+        instance._p_changed = True
+
+
+class Volume(zeit.content.article.edit.reference.Reference):
+
+    grok.implements(
+        zeit.content.article.edit.interfaces.IVolume)
+    type = 'volume'
+
+    references = ReferenceProperty('.', 'teaser_text')
```

Comment:
@wosc Meines Erachtens nach ist diese Stelle der Grund, weshalb man es nicht einfach in `reference.py` hinzufügen kann. Selbiges gilt für `author.py`. Ansonsten ist alles gleich, ja.